pallets/attestation/src/mock.rs

```diff
@@ -385,6 +387,13 @@ pub(crate) mod runtime {
 			ext
 		}
 
+		pub fn build_and_execute_with_sanity_tests(self, test: impl FnOnce()) {
+			self.build().execute_with(|| {
+				test();
+				Pallet::<Test>::do_try_state().unwrap();
```

Comment:
Better to use `expect(error_string)` then `unwrap()` in almost all circumstances 😃